app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -124,6 +134,7 @@ fun RecentlyVisited(
 @Composable
 private fun RecentlyVisitedHistoryGroup(
     recentVisit: RecentHistoryGroup,
+    enabled: Boolean,
```

Comment:
Can we move this before or after `showDividerLine`, just seems nicer to keep the Boolean arguments together?

Reply:
Sure!